code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -5549,3 +5549,108 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	reagent_state = SOLID
 	color = "#FFA500"
 	custom_metabolism = 0.1
+
+//Plant-specific reagents
+
+/datum/reagent/kelotane/tannic_acid
+	name = "Tannic acid"
+	id = TANNIC_ACID
+	description = "Tannic acid is a natural burn remedy."
+	reagent_state = LIQUID
+	color = "#150A03" //rgb: 21, 10, 3
+
+/datum/reagent/dermaline/kathalai
+	name = "Kathalai"
+	id = KATHALAI
+	description = "Kathalai is an exceptional natural burn remedy, it performs twice as well as tannic acid."
+	color = "#32BD08" //rgb: 50, 189, 8
+
+/datum/reagent/bicaridine/opium
+	name = "Opium"
+	id = OPIUM
+	description = "Opium is an exceptional natural analgesic."
+	color = "#AE9260" //rgb: 174, 146, 96
+
+/datum/reagent/space_drugs/mescaline
+	name = "Mescaline"
+	id = MESCALINE
+	description = "Known to cause mild hallucinations, mescaline is often used recreationally."
+	color = "#B8CD93" //rgb: 184, 205, 147
+
+/datum/reagent/synaptizine/nicotine
+	name = "Nicotine"
+	id = NICOTINE
+	description = "Nicotine is used to improve cognition, alertness, and focus."
+	color = "#A49B50" //rgb: 164, 155, 80
+
+/datum/reagent/hyperzine/cocaine
```

Comment:
You'll want to add this to the movement speed stuff so it actually works, but I'm going to hold off on doing anything with this PR until the movespeed stuff is merged so he doesn't keep having to fix conflicts.